composer.json

```diff
@@ -15,12 +15,13 @@
 	"require-dev": {
 		"phpunit/phpunit": "~4.5.1"
 	},
-	"suggests": {
+	"suggest": {
 		"ext-curl": "*"
 	},
 	"autoload": {
-		"psr-4": {
-			"Sofort\\SofortLib\\": "src"
-		}
+		"psr-0": {"Sofort\\SofortLib\\": "src"}
```

Comment:
I moved this to a psr-0. So I can autoload the test-sources by psr-4 without having any troubles in the autoloader.